contributor-guide/modules/ROOT/pages/modular-boost/establish-merge-point.adoc

```diff
@@ -0,0 +1,81 @@
+= Establishing a merge point after SVN Conversion
```

Comment:
I'm not sure this is relevant nowadays because this transition is already complete for any new contributors. I think we should at least have an admonition explaining this transition is already complete.

Reply:
If it is not needed, the section should be removed.

Reply:
Sounds good to me

Reply:
The section is still here